e2e-tests/src/driver.rs

```diff
@@ -303,16 +303,13 @@ impl Driver {
 
                 merod.run().await?;
 
-                let node_name = merod.name.clone();
-                e.insert(merod);
-
-                let merod_ref = &merods[&node_name];
+                let node_name = e.insert(merod);
```

Comment:
this variable name isn't descriptive of what is actually stored in the variable which is a `&mut Merod`

`merod` is more appropriate.

Reply:
Done.